pyproject.toml

```diff
@@ -181,10 +179,10 @@ openai = ["openai", "tiktoken"]
 text_helpers = ["chardet"]
 cohere = ["cohere"]
 in_memory_store = ["docarray"]
-hnswlib = ["docarray", "protobuf", "hnswlib"]
+hnswlib = ["docarray"]
```

Comment:
having `hnswlib` have the docarray dependency only seems weird. should call `docarray`?

Reply:
I thought that was strange too but had not opined here. I agree and will change.

Reply:
Added a "docarray" extra in https://github.com/hwchase17/langchain/pull/5015/commits/07a4ded643441e021e0c2634f1d60298f699f2b2